dialects/mysql/src/main/kotlin/app/cash/sqldelight/dialects/mysql/grammar/MySql.bnf

```diff
@@ -183,13 +202,13 @@ qualified_table_name ::= [ {database_name} '.' ] {table_name} [ index_hint_list
 index_hint_list ::= index_hint ( index_hint ) *
 
 index_hint ::=
-  ( 'USE' ( 'INDEX' | 'KEY' ) [ 'FOR' ( 'JOIN' | 'ORDER' 'BY' | 'GROUP' 'BY' )] [index_list] )
+  ( USE ( 'INDEX' | 'KEY' ) [ FOR ( JOIN | ORDER BY | GROUP BY )] [index_list] )
   |
-  ( ('IGNORE' | 'FORCE' ) ( 'INDEX' | 'KEY' ) [ 'FOR' ( 'JOIN' | 'ORDER' 'BY' | 'GROUP' 'BY' )] index_list ) {
+  ( (IGNORE | 'FORCE' ) ( INDEX | 'KEY' ) [ FOR ( JOIN | ORDER BY | GROUP BY )] index_list ) {
```

Comment:
```suggestion
  ( (IGNORE | 'FORCE' ) ( INDEX | KEY ) [ FOR ( JOIN | ORDER BY | GROUP BY )] index_list ) {
```

Reply:
@vanniktech Yeah, I do use a script now :D Find and replace was too annoying